README.md

```diff
@@ -18,21 +18,21 @@ Please refer to the [moose wiki](https://moosetechnology.github.io/moose-wiki/)
 
 ### Load Moose in a Pharo image
 
-**Stable version: Moose 8**
+**Latest version: Moose 9**
 
 ```smalltalk
 Metacello new
   baseline: 'Moose';
-  repository: 'github://moosetechnology/Moose:v8.0.0/src';
+  repository: 'github://moosetechnology/Moose:development/src';
   load.
 ```
 
-**Development version: Moose 9**
+**Oldish version: Moose 8**
```

Comment:
I'd like to keep `stable`.
I know we want people to use last version to give feedback, but I'm not telling this version is old is super nice since it is the only stable one